docs/content/docs/concepts/architecture.md

```diff
@@ -63,3 +63,18 @@ The Operator manages the lifecycle of Flink resources. The following chart illus
   - ROLLING_BACK : The resource is being rolled back to the last stable spec
   - ROLLED_BACK : The resource is deployed with the last stable spec
   - FAILED : The job terminally failed
+
+## Admission Control
+
+In addition to compiled-in admission plugins, a custom admission plugin named Flink Kubernetes Operator Webhook (Webhook)
+can be started as extension and run as webhook.
+
+The Webhook follow the Kubernetes principles, notably the [dynamic admission control](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+It's deployed by default when the Operator installed on a Kubernetes cluster using [Helm](https://helm.sh).
```

Comment:
nit: when the operator _is_ installed

Reply:
Thanks for the review, fixing it today at some point.

Reply:
Fixed.